ts/graphs/buttons.ts

```diff
@@ -140,9 +140,7 @@ export function renderButtons(
         const buttonTotal = data.count;
 
         const buttonTotalCommaSeparated = localizedNumber(buttonTotal);
-        const percent = buttonTotal
-            ? ((buttonTotal / groupTotal) * 100).toFixed(2)
-            : "0";
+        const percent = localizedNumber((buttonTotal / groupTotal) * 100);
```

Comment:
Removing the check here causes this to happen:
<img width="591" alt="Screen Shot 2021-12-28 at 10 47 25 am" src="https://user-images.githubusercontent.com/180542/147516227-a77472b8-a5f6-4c55-9329-b9fcdf29d38e.png">

Otherwise looks good!